desktop-widgets/CMakeLists.txt

```diff
@@ -141,3 +141,4 @@ add_library(subsurface_generated_ui STATIC ${SUBSURFACE_UI_HDRS})
 target_link_libraries(subsurface_generated_ui ${QT_LIBRARIES})
 add_library(subsurface_interface STATIC ${SUBSURFACE_INTERFACE} ${SUBSURFACE_UI_SRCS})
 target_link_libraries(subsurface_interface ${QT_LIBRARIES} ${GRANTLEE_LIBRARIES} subsurface_desktop_preferences)
+set_target_properties(subsurface_interface PROPERTIES CXX_STANDARD 14)
```

Comment:
ugh.
I'm not sure I want that.
What, specifically, can we do if we have that.
What does it break - e.g., what's the minimum gcc required (we build Windows with gcc 4.9 IIRC)